repo-client/src/main/java/io/lionweb/repoclient/api/HistoryAPIClient.java

```diff
@@ -0,0 +1,29 @@
+package io.lionweb.repoclient.api;
+
+import io.lionweb.lioncore.java.model.Node;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+
+public interface HistoryAPIClient {
+  @NotNull
+  List<Node> historyListPartitions(RepositoryVersionToken repoVersion) throws IOException;
```

Comment:
commonly, method names start with a verb.

Reply:
I removed the `history` prefix from this and the `historyRetrieve` (now `retrieve`) method.

Now they have the same name as the variants without history, but given Java supports method overloading and the type of the first parameter (`RepositoryVersionToken`) make it obvious which variants are connected with history, then I do not see a problem with that